client/db/bolt/db.go

```diff
@@ -509,17 +530,16 @@ func (db *BoltDB) MarketOrders(dex string, base, quote uint32, n int, since uint
 func (db *BoltDB) ActiveDEXOrders(dex string) ([]*dexdb.MetaOrder, error) {
 	dexB := []byte(dex)
 	return db.filteredOrders(func(oBkt *bbolt.Bucket) bool {
-		status := oBkt.Get(statusKey)
-		return bEqual(dexB, oBkt.Get(dexKey)) && (bEqual(status, byteEpoch) || bEqual(status, byteBooked))
-	})
+		return bEqual(dexB, oBkt.Get(dexKey))
+	}, false)
```

Comment:
No longer excluding unknown status. When would an order become status unknown? I assume is it unexpected?

Reply:
I think that's ok. Also, the insert/update order functions could be modified to prevent it from ever being a possibility.

Reply:
I believe all the update functions now error on unknown status.